engine/src/operators/OrderBy.h

```diff
@@ -18,6 +18,27 @@ bool is_sort(std::string query_part);
 
 void process_sort(blazing_frame & input, std::string query_part, Context * queryContext);
 
+}  // namespace operators
+}  // namespace ral
+
+
+
+namespace ral {
+namespace operators {
+namespace experimental {
+
+namespace {
+  using blazingdb::manager::experimental::Context;
+}  
+
+std::unique_ptr<ral::frame::BlazingTable> process_sort(const ral::frame::BlazingTableView & table, 
+    std::string query_part, Context & context);
+
+std::unique_ptr<ral::frame::BlazingTable>  distributed_sort(Context & context,
```

Comment:
this function should not exist. there is only one sort, it handles both single and distributed on the inside

Reply:
see my comments above regarding logic for single vs distriubted